backend/src/routes/v2/model/getModel.spec.ts

```diff
@@ -0,0 +1,21 @@
+// Some bad parts
+import '../../../utils/test/testUtils.js'
+
+import { describe, expect, test, vi } from 'vitest'
+
+import { createFixture, testGet } from '../../../utils/v2/test/routes.js'
+import { getModelSchema } from './getModel.js'
+
+describe('routes > model > getModel', () => {
```

Comment:
Tests for when parse (BadReq) and getModelByID (Forbidden, NotFound) throw errors?